tests/test_input.py

```diff
@@ -1751,7 +1789,7 @@ def test_learning_curve_objectives_and_no_metrics():
     eq_(grid_objectives, [])
 
 
-def test_learning_curve_default_objectives_and_no_metrics():
+def test_learning_curve_no_metrics():
```

Comment:
Shouldn't this test raise an error? Is there a default learning curve metric?

Reply:
This test is calling `check_config_parsing_value_error` which raises `ValueError`. There is no default metrics. Instead, I can change this test case to call `_parse_config_file` and raise exception if you think that would be better.